Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -177,8 +245,19 @@ public final class ExploreView: UIView {
 
 extension ExploreView: UICollectionViewDelegate {
     public func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
-        let item = sections[indexPath.section].items[indexPath.item]
-        delegate?.exploreView(self, didSelectItem: item, at: indexPath)
+        if indexPath.section < exploreSections.count {
+            let item = exploreSections[indexPath.section].items[indexPath.item]
+            delegate?.exploreView(self, didSelectItem: item, at: indexPath)
+        } else {
+            let item = recommendationsSection[indexPath.row]
+            delegate?.exploreViewRecommendations(self, didSelectItemAtIndex: indexPath.row, withId: item.id)
```

Comment:
This method needs to be renamed also, since it only applies to the last section. I'm guessing the order is that recommendations will end up in the last section, so can it be renamed to i.e.:

```swift
func exploreViewRecommendations(_ adRecommendationsGridView: ExploreView, didSelectRecommendationAt index: Int, withId: String)
```